content/eng-hiring/phone-screens.md

```diff
@@ -0,0 +1,194 @@
+
+2. **Screen out candidates who would not be successful.** Candidates who don't share our values, do not match the required skills for the position, or are interested in working on things or in ways which we cannot offer.
+
+Phone screens are a rough filter: it can't be used to really qualify a candidate, but it can be used to disqualify.
+
+**Because phone screens are decided upon by a single individual, phone screeners should be especially wary about their unconscious bias.**
+
+Please remember to take notes and pass them along: they can be very useful to future interviewers.
+
+## Before the call
+
+The candidate will have already received guidance from TTS Talent on what it’s like to interview at TTS and an explainer of payscales and term limits. This guide changes depending on the hiring authority used and the specifics of the role, but here’s an example [candidate guidance for a DHA role](https://docs.google.com/document/d/10n9DnWP07fjsRcM0Yw3z6OlbWi2nHcg0cKUhXIy-3po/edit) and for [a reserved hire role](https://docs.google.com/document/d/1ci_PgaJty6esQ_C0yk8DKP_8ySpdaxR9plVbK7AO95o/edit).
+
+Take a few minutes to familiarize yourself with the candidate’s resume, copy the notes template, and review what to look for during a phone screen.
+
+<a class="usa-button usa-button-primary" href="https://docs.google.com/document/d/1vMvOODdI_aRMPj6GpzkfY0KxmsUZLKCngun3ub6yJJY/copy">Make a copy</a> of [the notes template](https://docs.google.com/document/d/1vMvOODdI_aRMPj6GpzkfY0KxmsUZLKCngun3ub6yJJY/edit).
+
+## Conversation Outline
+
+### Introduce yourself and set expectations
+
+Begin by introducing yourself by saying this or something similar to it:
+
+> Hello! My name is \_\_\_, my pronouns are \_\_\_, and my role at 18F is \_\_\_. In a nutshell, 18F is a digital consultancy and we work with our clients, who we call our partners, to develop and modernize technologies in service of the American public.
+>
+> Thank you for taking the time to speak with me today! The purpose of this call is to make sure we’re on the same page about what 18F is and what the Consulting Software Engineer role entails, to ask a few questions to get to know you and your experience better, and to answer any questions that you may have.
+>
+> I’ll lead the conversation for about 45 minutes, and then hand the conversation off to you to answer any questions that you might have for the remaining 15 minutes.
+>
+> I’m excited that you’re here — are you ready to get started?
+
+### Discuss the candidate’s career history and path
+
+> Give the candidate space to tell their story, so we can hear how they view themselves and where they are heading in their career.
```

Comment:
For some reason this page has these block-quote/indents  for the first paragraph. It's like this in production as well. I don't understand the original purpose of it, but it looks a bit off. You can leave as is if our general strategy is "lift-and-shift" but can consider removing.

Reply:
You're right, it makes sense that the blockquotes should be for the questions, not the descriptions.